src/main/java/org/jabref/gui/sidepane/SidePaneContentFactory.java

```diff
@@ -66,7 +66,6 @@ public Node create(SidePaneType sidePaneType) {
             case OPEN_OFFICE -> new OpenOfficePanel(
                     tabContainer,
                     preferences,
-                    aiService,
```

Comment:
Remove the import, checkstyle happy

Reply:
Nvm, was just one line so i removed it

Reply:
Nope, seems like that was wrong. Reverting.

Reply:
Yep, should be fine now

Reply:
@subhramit thanks!